DKit.py

```diff
@@ -158,6 +158,10 @@ def __exit__(self, type, value, traceback):
         if not (server_process is None) and server_process.poll() is None:
             server_process.terminate()
 
+    def on_window_command(self, window, command_name, args):
+        if command_name == ("exit" or "close_window"):
```

Comment:
Okay, that makes more sense. The tuple was returning a bool, yet the code still ran correctly somehow. But, it makes more sense your way. Will push a fix.